cli/cmd/resources/managers.go

```diff
@@ -24,6 +25,10 @@ func CreateResourceManagers(client *kube.Client, odigosNs string, odigosTier com
 		resourceManagers = append(resourceManagers, odigospro.NewOdigosProResourceManager(client, odigosNs, config, odigosTier, proTierToken, managerOpts))
 	}
 
+	if managerOpts.IncludeCentralProxy {
```

Comment:
consider calculating `IncludeCentralProxy` from `config` so it will work for all commands and requires less code and maintaince